src/odb/include/odb/db.h

```diff
@@ -7509,7 +7651,15 @@ class dbModule : public dbObject
 
   static dbModule* getModule(dbBlock* block_, uint dbid_);
 
-  std::string getHierarchicalName() const;
+  unsigned getModInstCount();
+  unsigned getDbInstCount();
```

Comment:
Just use int.  From https://google.github.io/styleguide/cppguide.html#Integer_Types ` In particular, do not use unsigned types to say a number will never be negative.`

Reply:
Fixed,Interesting to see the style guide indicate this is an error in the definition of iterators.

Reply:
I think it is now considered a mistake but unchangeable

Reply:
From Stroustrup himself in https://open-std.org/jtc1/sc22/wg21/docs/papers/2019/p1428r0.pdf

`The original use of unsigned for the STL was a bad mistake and should be corrected (eventually).`